backend/src/schema.graphql

```diff
@@ -207,6 +210,10 @@ type Post {
   )
 }
 
+input _PostFilter {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-12T08:44:42Z" title="Wednesday, June 12th 2019, 10:44:42 am +02:00">Jun 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -207,6 +210,10 @@ type Post {
   )
 }
 
+input _PostFilter {
```


We don't have to redefine the `_PostFilter` on our side, that's done by the library `neo4j-graphql-js` when we augment the schema.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-12T08:46:12Z" title="Wednesday, June 12th 2019, 10:46:12 am +02:00">Jun 12, 2019</time>_

---


@Tirokk the library will only add types and inputs etc. if they haven't been defined yet, so defining it on our side will effectively disable this behaviour.